src/go/k8s/tests/e2e/additional-cmdline-arguments/00-assert.yaml

```diff
@@ -25,6 +17,4 @@ status:
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
 collectors:
-- type: pod
-  selector: app.kubernetes.io/name=redpanda
-  tail: -1
+- command: ../../../hack/get-redpanda-info.sh
```

Comment:
I've seen just `hack/...` instead of `../../../hack/...` elsewhere...

Reply:
I don't know why `collectors` are running from within test folder but `commands` are executed from the root of the `src/go/k8s`.

```
apiVersion: kuttl.dev/v1beta1
kind: TestAssert
collectors:
  - command: ../../../hack/get-redpanda-info.sh
```

```
apiVersion: kuttl.dev/v1beta1
kind: TestAssert
commands:
  - command: hack/verify-condition-message.sh centralized-configuration-bootstrap append_chunk_size
```